testgrid/config.yaml

```diff
@@ -2445,6 +2445,9 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-org-verify-all
   num_columns_recent: 30
 #presubmits
+- name: pull-cluster-api-bazel-verify
```

Comment:
this needs to be added to one of the dashboards (cluster lifecycle?)

Reply:
Done : )